bref

```diff
@@ -90,6 +90,12 @@ $app->command('init', function (SymfonyStyle $io) {
  */
 $app->command('cli function [--region=] [arguments]*', function (string $function, ?string $region, array $arguments, SymfonyStyle $io) {
     $lambda = new SimpleLambdaClient(($region ?: getenv('AWS_DEFAULT_REGION')) ?: 'us-east-1');
+    
+    // Because arguments may contain spaces, and are going to be executed remotely
+    // as a separate process, we need to escape all arguments.
+    $arguments = array_map(static function (string $arg): string {
+        return escapeshellarg($arg);
+    }, $arguments);
```

Comment:
This *should* work in all situations, because the quoting is lost when passed through Symfony's input layer. Since the invocation on the Lambda side does not use Symfony console, this should be correct. And it is definitely more secure.